code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -412,6 +412,18 @@
 			new /obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped/neaeracube(src)
 
 
+/obj/item/weapon/storage/box/wolpincubes
+	name = "wolpin cube box"
+	desc = "Drymate brand wolpin cubes. Just add water!"
+	icon = 'icons/obj/food.dmi'
+	icon_state = "monkeycubebox"
+	storage_slots = 7
+	can_hold = list("/obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped/wolpincube")
+	New()
+		..()
+		for(var/i = 1; i <= 5; i++)
+			new /obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped/wolpincube(src)

```

Comment:
That can be done in the future, but...outside of the scope of this PR, especially given literally everything else in this file is also relatively pathed.